### DISCUSSION TITLE
Changing AniMe Matrix Clock Format to 12 hour format?

### DISCUSSION THREAD

**Discussion by lilzoe5** [seerge/g-helper#137] [Ideas]
Is this at all possible for people who use AM/PM? Thanks!

**Comment by seerge** [comment]
Hello, i can add that - but then line will be longer, and it means we have to drop seconds (at all) or make font way smaller (less readable). What do you prefer? :)

**Reply by lilzoe5** [reply]
I think removing the seconds would be good, thank you!

**Reply by seerge** [reply]
It's added in https://github.com/seerge/g-helper/releases/tag/v0.41 :)
